pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -160,9 +160,14 @@ func (c *Cluster) Start() error {
 				return errors.Wrapf(err, "failed to create mgr deployment %s", resourceName)
 			}
 			logger.Infof("deployment for mgr %s already exists. updating if needed", resourceName)
-
-			if err := updateDeploymentAndWait(c.context, c.clusterInfo, d, config.MgrType, mgrConfig.DaemonID, c.spec.SkipUpgradeChecks, false); err != nil {
+			if err := updateDeploymentAndWait(c.context, c.clusterInfo, d, config.MgrType, mgrConfig.DaemonID, c.spec.SkipUpgradeChecks, c.spec.ContinueUpgradeAfterChecksEvenIfNotHealthy); err != nil {
 				logger.Errorf("failed to update mgr deployment %q. %v", resourceName, err)
+				if c.spec.ContinueUpgradeAfterChecksEvenIfNotHealthy {
+					logger.Infof("continuing reconcile because ContinueUpgradeAfterChecksEvenIfNotHealthy is true")
+				} else {
+					logger.Infof("stopping reconcile after error waiting for mgr  because ContinueUpgradeAfterChecksEvenIfNotHealthy is false")
```

Comment:
Nit, typo: extra space between words

```suggestion
					logger.Infof("stopping reconcile after error waiting for mgr because ContinueUpgradeAfterChecksEvenIfNotHealthy is false")
```